_includes/releases/cloud/2023-04-10.md

```diff
@@ -0,0 +1,25 @@
+## April 10, 2023
+
+<h3> General changes </h3>
+
+- Contract customers can now [create {{ site.data.products.dedicated }} advanced clusters](../cockroachcloud/create-your-cluster.html#step-1-start-the-cluster-creation-process), which have all the features of {{ site.data.products.dedicated }} standard clusters, plus security features needed for [PCI compliance](../cockroachcloud/pci-dss.html). To upgrade your organization to a contract, [contact us](https://cockroachlabs.com/contact-sales).
+- {{ site.data.products.dedicated }} [third party metrics integrations](../{{site.versions["stable"]}}/third-party-monitoring-tools.html) (Datadog and Amazon CloudWatch) will now [export the following metrics](../cockroachcloud/export-metrics.html) to third party targets:
+  - `storage_l0_sublevels`
+  - `storage_l0_num_files`
+  - `security_certificate_expiration_ca`
+  - `sql_mem_root_current`
+  - `sys_host_disk_iopsinprogress`
+  
+<h3> Cloud API changes </h3>
+
+- [Cloud API](../cockroachcloud/cloud-api.html) users can now upgrade their cluster's [CockroachDB version](index.html), as well as roll back or manually finalize pending upgrades.
+- The [Cloud API endpoints](../cockroachcloud/cloud-api.html) for managing metric export configurations have been replaced with endpoints specific to the third party tool being used. For example, `EnableMetricExport` is now replaced by `EnableDatadogMetricExport` and `EnableCloudWatchMetricExport`.
```

Comment:
Are the old points deprecated for a period of time or are they no longer available in the API? It seems like this could require code changes at the customer side.
```suggestion
- The [Cloud API endpoints](../cockroachcloud/cloud-api.html) for configuring metric export have been replaced with specific endpoints for each supported third-party integration. For example, `EnableMetricExport` is now replaced by `EnableDatadogMetricExport` and `EnableCloudWatchMetricExport`.
```